en_US/rule/backend_tablestore.md

```diff
@@ -0,0 +1,95 @@
+# 保存数据到 Alibaba Tablestore 数据库
+
+Create a database instance (please refer to the official Tablestore documentation for the steps to create an instance).
+
+</br>Click `Instances` . `Time Series Table`. `Create Time Series Table`.
+
+![img](./assets/rule-engine/tablestore/instance_manage.png)
+
+
+Manage Time Series Table
+
+Click on the `Time Series Table` created in the previous step to enter the timeline management.
+Click `Data Management`, `Query Data`, and `Add Time Series` according to the business design.
+(This step can be skipped, Tablestore will automatically create a new `Time Series` when writing data that does not exist, the example does not operate on `Time Series`)
+
+![img](./assets/rule-engine/tablestore/timeline_mamge.png)
+
+## Create rule engine resources
+
+Click `Rule Engine` `Resources` `Create`. Find the `Tablestore(Time Series)`.
+
+![img](./assets/rule-engine/tablestore/create_resource.png)
+
+Using the created instance properties, fill in the corresponding `Tablestore server` and `Instance` names.
+
+</br>
+AccessKey & Secret need to use Alibaba Cloud account application, detailed application steps, please refer to the official documentation of Tablestore.
+</br>
+Please fill in different domain names according to the deployment method for the server address.
+
+![img](./assets/rule-engine/tablestore/tablestore_data.png)
+
+## Create Rule
+
+![img](./assets/rule-engine/tablestore/create_rule.png)
+
+Rule SQL used in the example:
+
+```SQL
+SELECT
+
+payload.k_int as k_int,
+payload.k_bool as k_bool,
+payload.k_str as k_str,
+payload.k_bin as k_bin,
+payload.k_float as k_float,
+clientid
+
+FROM
+ "#"
+```
+
+### Create Action
+
+`Table Name` and `ata Source` name, please fill in the `Table Name` and `Data Source` name as created (`Data Source` name can be empty string).
+
+![img](./assets/rule-engine/tablestore/create_action.png)
+
+Parameter List
+| Parameter | Definition |
+| --- | --- |
+| Measurement | Tablestore Measurement |
+| Data Source | Tablestore Data Source, default is empty string |
+| TimeStamp(microsecond) |In microsecondMQTT messages in EMQX are timestamped with millisecond precision by default and cannot be used directly. Fill in the blank value,the default value is the microsecond timestamp of the message arriving at EMQX |
```

Comment:
I think the beginning `In microsecond` text is not supposed to be there? Perhaps should be deleted?